.github/workflows/docker-images/yocto-builder/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:20.04
+
+ARG DEBIAN_FRONTEND="noninteractive"
```

Comment:
Why ARG and not ENV?

Reply:
Because there is no need to pass these after building the image.